pkg/queue/stats_reporter.go

```diff
@@ -30,15 +30,16 @@ import (
 type Measurement int
 
 const (
-	// ReportingPeriod interval of time for reporting.
-	ReportingPeriod = 10 * time.Second
+	// ViewReportingPeriod is the interval of time between reporting aggregated views.
+	ViewReportingPeriod = time.Second
+	// ReporterReportingPeriod is the interval of time between reporting stats by queue proxy.
+	// It should be equal to or larger than ViewReportingPeriod so that no stat
+	// will be dropped if LastValue aggregation is used for a view.
+	ReporterReportingPeriod = time.Second
 
-	// OperationsPerSecondN
-	OperationsPerSecondN = "operations_per_second"
-	// AverageConcurrentRequestsN
-	AverageConcurrentRequestsN = "average_concurrent_requests"
-	// LameDuckN
-	LameDuckN = "lame_duck"
+	operationsPerSecondN       = "operations_per_second"
+	averageConcurrentRequestsN = "average_concurrent_requests"
+	lameDuckN                  = "lame_duck"
```

Comment:
Nice that you unexported theses, when I first developed this code I was having trouble figuring out how to use them in the tests which is why I had exported them to begin with. However if all is good now then I'm fine with this.